Interfaces/scripts/dirac-wms-job-parameters.py

```diff
@@ -37,8 +37,6 @@
   if not result['OK']:
     errorList.append((job, result['Message']))
     exitCode = 2
-  else:
```

Comment:
Why removing this? I think it's needed...

Reply:
No because there is a `printOutput=True` in the call that does exactly that IIRC

Reply:
This is what we get currently:
```
[lxplus602] ~ $ dirac-wms-job-parameters 412766509
{412766509L: {'CPU(MHz)': '1400.000',
              'CPUNormalizationFactor': '3.1',
              'CPUScalingFactor': '3.1',
              'CacheSize(kB)': '1024KB',
              'HostName': 'r111c07s01',
              'JobWrapperPID': '8859',
              'LocalAccount': 'a07lhc01',
              'Memory(kB)': '0kB',
              'ModelName': 'Intel(R)XeonPhi(TM)CPU7250@1.40GHz',
              'OK': 'True',
              'PilotAgent': 'v10r0p15',
              'Pilot_Reference': 'htcondorce://r000u11l06-fe.marconi.cineca.it/1346248.0'}}
{'CacheSize(kB)': '1024KB', 'ModelName': 'Intel(R)XeonPhi(TM)CPU7250@1.40GHz', 'JobWrapperPID': '8859', 'LocalAccount': 'a07lhc01', 'Memory(kB)': '0kB', 'HostName': 'r111c07s01', 'CPUScalingFactor': '3.1', 'Pilot_Reference': 'htcondorce://r000u11l06-fe.marconi.cineca.it/1346248.0', 'CPU(MHz)': '1400.000', 'PilotAgent': 'v10r0p15', 'CPUNormalizationFactor': '3.1', 'OK': 'True'}
```